web-console/3-project-details/labels.md

```diff
@@ -0,0 +1,71 @@
+# Labels on Detail Pages
+
+## View Labels
+- Labels are currently displayed at the top of most detail pages (Deployments, Pods, Builds, Images, Storage, etc)
+- A new `Labels` section should be added to the bottom of the `Details` tab on all of these pages to allow for a more comprehensive labels area
+
+![Label 1](img/new-labels-00-copy.png)
+
+## Add Labels
+![Label 1](img/new-labels-01-copy.png)
+
+- An `Add` link should be displayed below the applied labels
+- Clicking the link will open an overlay panel with a combo box and a text field for entering label Name and Value.
+
+![Label 1](img/new-labels-03-copy.png)
+
+- Once Name and Value exist, the Add icon should change from the disabled state to a primary button color.
```

Comment:
Just curious, why not a button here instead of the "+"? Or icon plus text as you have on the edit modal?

Reply:
Good question. I definitely could add the text in if you think that helps! Originally I had a save button but it didn't make sense since this isn't a modal. When the user clicks add, the new label should appear on the page, while the panel is still open and allows users to continue adding as many as they would like. The Edit modal is slightly different since it is a true modal so the "+ Add" function there only adds a new row. Clicking save actually adds the label to the page. 

So with that said, do you think the add label and the add row functions should just both use the "+ Add"  syntax? I'm torn on this one!

Reply:
I’m wondering now if you need the separate Add Labels action on the details page since you can also add from the Edit modal. What if the action was Add or Edit Labels and you just use the edit modal you designed?

Reply:
Add row and add label are the same action aren't they?

Reply:
@ncameronbritt good point, I've debated that as well. I thought it was cleaner to have a quick add and then the more complex modal for edit/remove but I can see possibly combining them and just having "Edit" available on detail pages. 

I thought the summary pages benefited from having the add and edit separate to denote when users have selected objects with the same labels and edit was available, vs when they are adding labels at any time.